dialogues/risawoz/src/knowledgebase/en_zh_mapping.py

```diff
@@ -0,0 +1,141 @@
+class RisaWOZMapping(object):
+    def __init__(self):
+        # currently untranslated
+        # TODO: this mapping should only include slots that are required to make an api call for the domain. It should not include all the slots.
```

Comment:
This is an important TODO. In BiToD, they have this concept of `required_slots` which is defined as slots needed to be filled out before agent can make an api call. These are a subset of all slots available to a domain. Can we define that mapping for Risawoz?

Reply:
The required slots can be found in the original RiSAWOZ paper. I use basically the same translation in the paper for slots so they can be directly used.